06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,264 @@
+const uniqueTermsBook = ['alice', 'falls', 'into', 'a', 'rabbit'];
+const books = require('../assets/books.json');
+const wrongdata = require('../assets/wrongdata.json');
+const emptyfile = require('../assets/emptyfile.json');
+const smallfile = require('../assets/smallfile.json');
+
+describe('InvertedIndex Class', () => {
+  beforeAll(() => {
+    this.invertedIndex = new InvertedIndex();
+  });
+
+  describe('Class instantaion', () => {
+    it('should be instantiated with the "new" keyword', () => {
+      const init = () => {
+        InvertedIndex();
+      };
+      expect(init)
+        .toThrowError('Class constructor ' +
+          'InvertedIndex cannot be invoked without \'new\'');
```

Comment:
Thank you for pointing this out, I have used template literals to rectify this issue. thanks.